src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const order of actions) {
+    switch (true) {
+      case order.type === 'addProperties':
```

Comment:
why `switch (true)`

Reply:
Good advice !  There is a possibility to make switch more obvious.